bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -210,15 +214,18 @@ end
 
 ```javascript
 rules.when().item('Apartment_Window').changed().to('OPEN').then(() => {
-  actions.notificationBuilder('Apartment window was opened!').withIcon('window').withSeverity('HIGH').send();
+  actions.notificationBuilder('Apartment window was opened!')
+          .withIcon('window')
```

Comment:
Is the indentation correct here? In the line above, two spaces are used. So with eight spaces in this line, three levels deeper?